TGC/events/ARGFlavor.txt

```diff
@@ -1708,214 +1710,214 @@
-				random_country = {
-					limit = { exists = yes tag = CRT vassal_of = ARC ai = yes }
-					annex_to = ENT
-					CRT = { all_core = { remove_core = CRT add_core = ENT } }
-				}
-				any_country = {
-					limit = { war_with = THIS }
-					end_war = THIS
-				}
-				change_tag_no_core_switch = ARC
-
-				ai_chance = { factor = 0 }
-			}
-		}
-
-		#Description: The Argentine Confederation - Countries can choose whether to support Entre Rios or not
-		country_event = {
-			id = 46605
-			title = "EVTNAME46605"
-			desc = "EVTDESC46605"
-			picture = "rosas_urquiza"
-
-			trigger = {
-				ENT = {
-					exists = yes
-					has_country_flag = urquiza_statement
-				}
-				ARG = { exists = no }
-				NOT = {
-					OR = {
-						tag = ARC
-						tag = CRT
-						tag = SBA
-						tag = ENT
-						tag = ARG
-						tag = LPL
-					}
-					has_country_flag = will_support_entre_rios
-					has_country_flag = will_not_support_entre_rios
-					has_global_flag = entre_rios_rebellion
-				}
-				OR = {
-					any_owned_province = {
-						any_neighbor_province = {
-							OR = {
-								region = ARG_2360 # Entre Rios
-								province_id = 2359 # Colon
-							}
-						}
-					}
-					AND = {
-						OR = {
-							neighbour = ARC
-							neighbour = CRT
-							neighbour = ENT
-							AND = {
-								tag = URU
-								ruling_party_ideology = liberal
-							}
-						}
-						capital_scope = { continent = south_america }
-						NOT = { relation = { who = ARC value = 100 } }
-					}
-				}
-				OR = {
-					has_country_flag = democracy_gov
-					has_country_flag = constitutional_monarchy_gov
-				}
-				ARC = {
-					government = presidential_dictatorship
-					NOT = { has_country_flag = entre_rios_has_happened }
-				}
-			}
-
-			mean_time_to_happen = {
-				days = 5
-			}
-
-			# Support Entre Rios
-			option = {
-				name = "EVTOPTA46605"
-
-				set_country_flag = will_support_entre_rios
-				any_pop = {
-					scaled_militancy = {
-						factor = -3
-						ideology = liberal
-					}
-				}
-				relation = {
-					who = ARC
-					value = -150
-				}
-				leave_alliance = ARC
-
-				ai_chance = {
-					factor = 80
-					modifier = {
-						factor = 0.75
-						alliance_with = ARC
-					}
-					modifier = {
-						factor = 0.5
-						num_of_revolts = 1
-					}
-					modifier = {
-						factor = 0.9
-						tag = URU
-						ruling_party_ideology = liberal
-					}
-				}
-			}
-
-			# Don't support Entre Rios
-			option = {
-				name = "EVTOPTB46605"
-
-				set_country_flag = will_not_support_entre_rios
-				any_pop = {
-					scaled_militancy = {
-						factor = 3
-						ideology = liberal
-					}
-				}
-
-				ai_chance = {
-					factor = 20
-					modifier = {
-						factor = 0.5
-						truce_with = ARC
-					}
-					modifier = {
-						factor = 0
-						NOT = { relation = { who = ARC value = 0 } }
-					}
-				}
-			}
-		}
-
+#		#Description: Urquiza's Statement - issue a statement of independence if Rosas doesn't step down
```

Comment:
Is this intentionally disabled?

Reply:
ARG content was intentionally disabled until the ARC rework is finished